backend/src/main/java/ai/verta/modeldb/artifactStore/storageservice/S3Service.java

```diff
@@ -94,11 +100,32 @@ public String generatePresignedUrl(String s3Key, String method, long partNumber,
         new GeneratePresignedUrlRequest(bucketName, s3Key)
             .withMethod(reqMethod)
             .withExpiration(expiration);
-    if (partNumber != 0) {
-      request.addRequestParameter("partNumber", String.valueOf(partNumber));
-      request.addRequestParameter("uploadId", uploadId);
-    }
+    request.addRequestParameter("partNumber", String.valueOf(partNumber));
+    request.addRequestParameter("uploadId", uploadId);
```

Comment:
@evgeniilezhnin What was the reason for this change?
Based on my testing, this adds `"uploadId=&partNumber=0"` to the presigned URL when Client sends `part_number: 0`, which causes an error from S3 during upload.

I recommend reverting

Reply:
There wasn't this check in feature/VR-4346_log_initialize_multipart , after that I added it, but forgot to add it in feature/VR-4348_commit_artifact_part . So this change is not needed here, I'll remove it

Reply:
done